lib/dynflow/executors.rb

```diff
@@ -8,9 +8,10 @@ module Executors
     # we should wrap it with this method, and we can ensure here to do
     # necessary cleanup, such as cleaning ActiveRecord connections
     def self.run_user_code
+      clear_connections = defined?(::ActiveRecord) && ActiveRecord::Base.connection.open_transactions.zero?
```

Comment:
Tests are failing, probably because we actually don't set the connection. Adding `ActiveRecord::Base.connected?` should help.